ads/google/doubleclick.md

```diff
@@ -16,116 +16,4 @@ limitations under the License.
-
-To request an ad with multiple sizes, pass a string of comma-separated sizes to
-the `data-multi-size` attribute. Each size in the list must be a width followed
-by a lowercase 'x' followed by a height. Secondary sizes must not be larger than
-their corresponding dimensions specified by the `width` and `height` attributes,
-or the `data-override-width` and `data-override-height` attributes, if they are
-set. Further, the secondary sizes must not be smaller than 2/3rds, in any of the
-two dimensions, of their primary size counterpart, unless 
-`data-multi-size-validation` is explicitly set to false.
-
-Examples:
-
-#### With multi-size request
-```html
-<amp-ad width=728 height=90
-    type="doubleclick"
-    data-slot="/6355419/Travel"
-    data-multi-size="700x90,700x60,500x60">
-</amp-ad>
-```
-
-#### With multi-size request ignoring size validation
-```html
-<amp-ad width=728 height=90
-    type="doubleclick"
-    data-slot="/6355419/Travel"
-    data-multi-size="300x25"
-    data-multi-size-validation="false">
-</amp-ad>
-```
-
-
-### Supported parameters
-
-- `data-slot`
-- `data-multi-size`
-- `data-multi-size-validation`
-
-Supported via `json` attribute:
-
-- `categoryExclusions`
-- `cookieOptions`
-- `tagForChildDirectedTreatment`
-- `targeting`
-- `useSameDomainRenderingUntilDeprecated`
-
-### Temporary use of useSameDomainRenderingUntilDeprecated until March 29, 2018
-Support for this attribute will be dropped on March 29, 2018. 
-An experiment to use the higher performance GPT Light tag in place of the DoubleClick GPT tag causes the ad to render in a second cross domain iframe within the outer AMP iframe. This prevents ads from accessing the iframe sandbox information and methods which are provided by the AMP runtime. Until this API is available to work in the second level iframe, publishers can opt out of this experiment by including "useSameDomainRenderingUntilDeprecated": 1 as a json attribute. This attribute will be deprecated on March 29, 2018. After that point, the GPT Light tag will become the default and all eligible ads will always be rendered inside a second cross domain iframe. For more information, please refer to https://github.com/ampproject/amphtml/issues/11834;
-
-Example:
-```html
-<amp-ad width=320 height=50
-    type="doubleclick"
-    data-slot="/4119129/mobile_ad_banner"
-    json='{"useSameDomainRenderingUntilDeprecated":1}'>
-</amp-ad>
-```
-
-
-### Unsupported DFP Formats
-- Interstitials
-- Expandables. Although expandables on interaction/click is a format that is work in progress.
-- Flash
-- Anchor Ads / Adhesion Units
-- Creatives served over HTTP.
-
-
-
-
-
+Support for DoubleClick Delayed Fetch has been deprecated as of March 29, 2018. Please upgrade to Fast Fetch when possible. Visit the <a href="https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md">DoubleClick Fast Fetch documentation page</a> for more details.
```

Comment:
Since this is now in the past, what about:

`Support for DoubleClick Delayed Fetch has been removed as of March 29, 2018. Please upgrade to Fast Fetch.`